syntax = "proto3";

package  fixbug;

option  cc_generic_services = true;

message  ResaultCode
{
    int32     errcode = 1;
    bytes     errmsg =2;
}


message     LoginRequest
{
    bytes  name = 1;
    bytes   pwd = 2;
}

message     LoginResponse
{
    ResaultCode     result = 1;
    bool            success = 2;
}


service     UserServices
{
    rpc    Login(LoginRequest)      returns(LoginResponse);
}